outputs.tf

```diff
@@ -1,10 +1,35 @@
 output "groups" {
-  description = "A nested map of group resources, with group name as the key, and each value map containing the groups's ARN and Unique ID."
-  value = {
-    for group, properties in aws_iam_group.this : group => {
-      arn : properties.arn
-      name : properties.name
-      unique_id : properties.unique_id
+  description = "List of group objects with keys 'arn', 'name' and 'unique_id'"
+  value = { for x in concat([for group, properties in aws_iam_group.this : {
+    name : group
+    arn : properties.arn
+    unique_id : properties.unique_id
+    }
+    ], [{
+      name : aws_iam_group.admins.name
+      arn : aws_iam_group.admins.arn
+      unique_id : aws_iam_group.admins.unique_id
+    },
+    {
+      name : aws_iam_group.users.name
+      arn : aws_iam_group.users.arn
+      unique_id : aws_iam_group.users.unique_id
+    }
+  ]) : x.name => x }
+
+}
+
+output "users" {
+  description = "List of users objects with keys 'arn', 'name' and 'unique_id'"
+  value = { for user, properties in aws_iam_user.this : user => {
```

Comment:
I think we want `{ for user in aws_iam_user.this : user.name = {...}}`.

Reply:
This works as intended.